### ISSUE TITLE
Missing LambdaLayer in eu-central-2 

### ISSUE THREAD

**Issue by simonmesserli** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#867]
hi team,

The object cid-0.3.6.zip or even the s3 bucket is missing in eu-central-2. 

The creation of the CidLambdaLayer in cid-cdn.yml fails. 

aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-central-2/cid-resource-lambda-layer/cid-0.3.6.zip .
--> fatal error: An error occurred (403) when calling the HeadObject operation: Forbidden

can you please provide the zip for the layer in a S3 bucket in eu-central-2 (zurich)? 

thank you & cheers

Simon

**Comment by iakov-aws** [comment]
we are working on this but it will take some time. please use another region

**Comment by dscorourke** [comment]
This is either broken globally or the URI the CFN template is constructing is not valid. I have tried the 3 most recent versions of the layer, none of which work. All of them produce this error:
fatal error: An error occurred (403) when calling the HeadObject operation: Forbidden

Commands tried:
`aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-east-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-east-2/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-west-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-west-2/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-south-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-northeast-3/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-northeast-2/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-southeast-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-southeast-2/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-northeast-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ca-central-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-central-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-west-1/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-west-2/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-west-3/cid-resource-lambda-layer/cid-0.3.8.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-north-1/cid-resource-lambda-layer/cid-0.3.8.zip ./


aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-east-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-east-2/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-west-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-us-west-2/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-south-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-northeast-3/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-northeast-2/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-southeast-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-southeast-2/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ap-northeast-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-ca-central-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-central-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-west-1/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-west-2/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-west-3/cid-resource-lambda-layer/cid-0.3.7.zip ./
aws s3 cp s3://aws-managed-cost-intelligence-dashboards-eu-north-1/cid-resource-lambda-layer/cid-0.3.7.zip ./`

**Comment by dscorourke** [comment]
You're right, it turns out this was unrelated to the issue here. I have removed my previous comment to avoid confusing this issue.

**Comment by simonmesserli** [comment]
any updates to this?
